cpu/efm32/periph/i2c.c

```diff
@@ -66,18 +58,30 @@ static void _transfer(i2c_t dev, I2C_TransferSeq_TypeDef *transfer)
 
         irq_restore(cpsr);
     }
+
+    /* transfer finished, interpret the result */
+    switch (i2c_progress[dev]) {
+        case i2cTransferDone:
+            return I2C_ACK;
```

Comment:
We were wondering last week if it's better to return I2C_ACK when everything is fine or the number of bytes written on the bus.
What do you think ?

Reply:
I don't think it really matters. `I2C_ACK` would imply all bytes are transferred (either direction), otherwise you would receive a different error (e.g. bus arbitration lost).

Reply:
I agree but we need to get a consensus here so all CPUs will behave in the same way :)